machine_management/control_plane_machine_management/cpmso-using.adoc

```diff
@@ -117,4 +118,12 @@ include::modules/machineset-gcp-shielded-vms.adoc[leveloffset=+2]
 ** link:https://cloud.google.com/compute/shielded-vm/docs/shielded-vm#integrity-monitoring[Integrity monitoring]
 
 //Enabling customer-managed encryption keys for a machine set
-include::modules/machineset-gcp-enabling-customer-managed-encryption.adoc[leveloffset=+2]
\ No newline at end of file
+include::modules/machineset-gcp-enabling-customer-managed-encryption.adoc[leveloffset=+2]
+
+[id="cpmso-supported-features-openstack_{context}"]
+== Enabling OpenStack features for control plane machines
```

Comment:
I know you copy/pasted it from other platforms but I'm not sure in our case we can call it "Features". The link below is about changing a Nova flavor for the control plane. It's called vertical scaling, I don't think it can be described as "enabling OpenStack features"...

Reply:
I followed this comment: "[Managing control plane machines with control plane machine sets](https://docs.openshift.com/container-platform/4.13/machine_management/control_plane_machine_management/cpmso-using.html): add a section named "Enabling OpenStack features for control plane machines". At least one feature can be documented: the Nova Flavor can be changed (see AWS instance type, same concept, except the field name is "flavor")." 

Can you let me know the features that needs to be added here?

Reply:
Look at https://docs.openshift.com/container-platform/4.13/machine_management/control_plane_machine_management/cpmso-using.html#cpmso-supported-features-gcp_cpmso-using
We could document that a control plane can use `rootVolume` to store the data into a Cinder volume since it's part of the CPMS Failure Domain API.

However, due to performance issue we don't recommend doing it for now.

cc @mandre do you agree that we shouldn't advertise for this "feature"?

Other than that, I see nothing else.

Reply:
I think in the case of the OSP platform, we don't have feature to activate via CPMS/FailureDomains.
Yes it's possible to have rootVolumes added on day 2, but with all the Ceph performance issues when using volumes I'll not recommend that for now. We can still make a change later if we change our mind.

Reply:
@EmilienM +1. 

@subhtk So, looks like the sum of this is to reconsider the section heading.

Reply:
Yes. and the heading should be? Any suggestions?

Reply:
@maxwelldb Any suggestions for the heading? and Should I keep the feature or do I have to remove it?

Reply:
I'd check with @jeana-redhat and see if keeping the heading pattern is important or not.

Reply:
replicating https://github.com/openshift/openshift-docs/pull/64454#discussion_r1357119156

I don't have any problem with diverging from the pattern here to make the statement accurate. How do folks feels about "Updating the configuration for {rh-openstack} control plane machines"?

Reply:
That works well.